internal/controller/volsync/vshandler_test.go

```diff
@@ -689,12 +689,20 @@ var _ = Describe("VolSync_Handler", func() {
 					It("Should return a replication source and a RS should be created", func() {
 						// Run another reconcile - we have the psk secret now but the pvc is not in use by
 						// a running pod
+
+						var err error
 						finalSyncCompl, rs, err := vsHandler.ReconcileRS(rsSpec, false)
 						Expect(err).ToNot(HaveOccurred())
 						Expect(finalSyncCompl).To(BeFalse())
 						Expect(rs).ToNot(BeNil())
```

Comment:
In a the previous version you ran vsHandler.ReconcileRS(rsSpec, false) multiple times until it succeeded or timed out. Why did you change this in this version?